dask/utils.py

```diff
@@ -85,13 +140,9 @@ def ndeepmap(n, func, seq):
         return func(seq)
 
 
+@_deprecated(version="2021.06.1", use_instead="contextlib.supress")
```

Comment:
Another option here would be to do something like:

```python
ignoring = _deprecated(suppress, ...)
```

This is nice and compact, but requires us to provide an alternative function with the same signature for users to migrate to, which we may not always have. This is why I took the current approach which is more verbose, but also more generic.

Reply:
Yeah I like the approach you've taken. Plus having the original function around will motivate us to actually remove it at some point in the future.